src/plugins/recordTypes/intake/advancedSearch.js

```diff
@@ -0,0 +1,59 @@
+export default (configContext) => {
+  const {
+    OP_EQ,
+    OP_CONTAIN,
+    OP_RANGE,
+  } = configContext.searchOperators;
+
+  const {
+    defaultAdvancedSearchBooleanOp,
+    extensions,
+  } = configContext.config;
+
+  return {
```

Comment:
Please order these search fields into the same tab order as they appear on the record editor. That's what we do by default unless another order is requested. It doesn't look like pahma requested anything different for intake.